docs/included.md

```diff
@@ -312,7 +392,7 @@
   - 
     [Wikipedia](https://en.wikipedia.org/wiki/Sensitivity_and_specificity)
 * -  
-    - Success Ratio
+    - Success Ratio (Precision)
```

Comment:
Not sure that this matters. Precision has a different reference ```[Wikipedia](https://en.wikipedia.org/wiki/Precision_and_recall)```

Should they:
- Have the same reference,
- Both entries list both references, or
- Have different references?

Reply:
See discussion above